stan/math/rev/fun/exp.hpp

```diff
@@ -9,6 +9,7 @@
 #include <stan/math/prim/fun/isinf.hpp>
 #include <stan/math/prim/fun/isfinite.hpp>
 #include <stan/math/rev/fun/is_nan.hpp>
+#include <stan/math/rev/fun/cos.hpp>
```

Comment:
Stray include?

Reply:
Suprisingly no! This actually is used by the complex `exp()`, which before was being included through some one of the other files but now needs manually included